pkg/binding/bindingpolicy-resolver.go

```diff
@@ -124,7 +124,7 @@ type BindingPolicyResolver interface {
 	// GetSingletonReportedStateRequestsForBinding calls GetSingletonReportedStateRequestForObject
 	// for each of workload objects in the resolution if the resolution exists.
 	// If the resolution doesn't exist, empty slices are returned.
```

Comment:
We may also change the comment accordingly with the code change, regarding the returned result (a single slice and could be nil).

Reply:
Thanks, updated.